prompts/deepset-question-generation copy.yaml

```diff
@@ -0,0 +1,11 @@
+name: deepset/summarization
```

Comment:
Some leftover, not properly named YAML file I suppose.

Reply:
Woops! Sorry, merged the fix